src/templates/layouts/healthCareLocalFacility/FacilitySocialLinks.tsx

```diff
@@ -1,13 +1,8 @@
+import { FieldLink } from '@/types/drupal/field_type'
 import React from 'react'
 
-interface SocialLink {
-  uri: string
-  title: string
-}
-
-interface OperatingStatus {
-  url: string
-}
+type SocialLink = Pick<FieldLink, 'uri' | 'title'>
+type OperatingStatus = Pick<FieldLink, 'url'>
```

Comment:
Updated these types because TypeScript was made about my formatter function for `FieldLink` not having nullable `url` values. I looked at the local facility formatter, and it looks like the reason it didn't catch it is because the `field_operating_status` is `any` in that formatter.

<img width="543" alt="field_operating_status is any type" src="https://github.com/user-attachments/assets/cc0834b4-7202-4196-a344-78d07b4799a2" />